lib/litmus.ex

```diff
@@ -1,18 +1,44 @@
 defmodule Litmus do
   @moduledoc """
-  Documentation for Litmus.
+  Documentation for Litmus. Validate data against schema defined.
   """
 
+  alias Litmus.Validation.{Params, Schema}
+
   @doc """
-  Hello world.
+  Validate your data against a schema
 
   ## Examples
 
-      iex> Litmus.hello
-      :world
+      iex(1)> alias Litmus.Type
+      Litmus.Type
```

Comment:
I'm thinking we should leave this alias out and just use `Litmus.Type.Any/String`?